yaml_files/simphony_metadata.yml

```diff
@@ -418,5 +418,36 @@ CUDS_KEYS:
     definition: Multiphase model
 
   ELECTROSTATIC_MODEL:
-    parent: CUBA.PHYSICS_EQUATION
     definition: Electrostatic model
+    parent: CUBA.PHYSICS_EQUATION
+    models: [CUBA.CFD, CUBA.MOLECULAR_DYNAMICS, CUBA.DEM]
+
+  BASIS:
+    definition: Space basis vectors
+    parent: CUBA.CUDS_COMPONENT
+    CUBA.VECTOR:
+      shape: (3, 3)
```

Comment:
The default value is `[[1, 0, 0], [0, 1, 0], [0, 0, 1]]` which is of shape (3, 3).
Or do we only check the first dimension?